### ISSUE TITLE
Unable to change mode from Mobile view to Desktop view in any browser

### ISSUE THREAD

**Issue by aruns501** [microsoft/vsts-extension-retrospectives#349]
I was exploring the possibility of changing the views as given for this plugin today. I noticed that while I wasa able to successfully change from Desktop to Mobile view, am unable to change back i.e. revert to existing from the changed view as I do not see the options appear.

Also in the MObile view am unable to create a new retrospective or edit any. In fact none off the manage options appear. This happened at around 1.00 PM IST today.
However when I accessed to check now I am able to see the regular desktop view, while I was unable to see in this view after multiple refresh actions taken from 1 - 2.00 PM IST today.

1. Can you please explain how one can change view from Mobile to Desktop?
2. Can you also explain why this behaviour auto changed after few hours -- is there any timing ?

Thanks
SA Kumar

**Comment by bastoneking** [comment]
I too noticed that the menu (top right of screen) is no longer available when switching to mobile view. When I refresh to page, I am returned to the "desktop" view.

Also, when clicking the menu, I am also given the option to "Switch to Desktop View". Selecting this option changes to the "mobile" view presumably b/c I am already in "desktop" view. Should this even be an option to user that is already in a desktop view?

**Comment by dphanson63** [comment]
Fixed in recent release.
